mettle/src/process.c

```diff
@@ -9,6 +9,7 @@
 #include <signal.h>
 #include <string.h>
 #include <unistd.h>
+#include <pty.h>
```

Comment:
Looks like we'll want `util.h` with guard clauses
```
#ifdef __linux__
...
```

Reply:
I think best practice _might_ be including your own pty.h, and using this `include_next` pattern?

https://github.com/rapid7/mettle/blob/a034d411907bcebbd610ae57db58347586742f96/mettle/include/compat/time.h#L6-L14